ocaml/xapi/xapi_pool.ml

```diff
@@ -3404,12 +3404,12 @@ let check_update_readiness ~__context ~self ~requires_reboot =
 
 let enable_client_certificate_auth ~__context ~self ~name =
   (* TODO: actually enable client-certificate auth *)
```

Comment:
in a future PR you will want to do an stunnel restart, rather than a reload? (because reloading does not reload the config)

restarting would of course drop the connection, so it's not ideal

Reply:
I believe an Stunnel reload (via systemd which is sending a signal) does reload the configuration.

Reply:
Yes, I think that's the only option.

Reply:
OK, I'll try it out.